cdap-master/pom.xml

```diff
@@ -165,6 +165,9 @@
         <package.depends>--depends cdap --depends cdap-hbase-compat-0.96 --depends cdap-hbase-compat-0.98
         --depends cdap-hbase-compat-1.0 --depends cdap-hbase-compat-1.0-cdh
         --depends cdap-hbase-compat-1.1</package.depends>
+        <stage.artifacts.dir>${stage.opt.dir}/artifacts</stage.artifacts.dir>
+        <additional.artifacts.jar.pattern>**/target/*.jar</additional.artifacts.jar.pattern>
+        <additional.artifacts.config.pattern>**/target/*.json</additional.artifacts.config.pattern>
```

Comment:
nope this picks up the file that could contain the doc properties built by hydrator-plugins